### DISCUSSION TITLE
Adding custom values to tfevents

### DISCUSSION THREAD

**Discussion by ninafiona** [facebookresearch/detectron2#4127] [Q&A]
I use a custom version of COCOEvaluator and run it every 250th iteration during training. I calculate custom IoU, precision and recall values and print them to my terminal/log-file. Now I would like to save them to my TF event files to visualize them in tensorboard as well. How can I achieve this? 

P.S. I also added this custom tensorboard writer to visualize validation loss: https://eidos-ai.medium.com/training-on-detectron2-with-a-validation-set-and-plot-loss-on-it-to-avoid-overfitting-6449418fbf4e
